tests/unittest/_torch/thop/parallel_hw_agnostic/test_noaux_tc.py

```
@@ -44,8 +45,7 @@ def test_noaux_tc_run(seq_len, num_experts, n_group, topk_group, top_k, dtype):
     gate.load_weights([weights])
     gate.cuda()
     with torch.inference_mode():
-        selected_indices, selected_values = gate.routing_method.apply(
-            gate.forward(logits))
+        selected_indices, selected_values = gate.routing_method.apply(gate.forward(logits))
```

Ruff (0.15.12)

[warning] Unpacked variable `selected_indices` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)